catroid/res/menu/menu_current_project.xml

```diff
@@ -43,7 +43,7 @@
     <item
         android:id="@+id/delete"
         android:icon="@drawable/ic_menu_delete"
```

Comment:
It would make more sense to remove the drawable altogether as you won't need an icon for a menu entry (same for the other three occurrences of this line and the drawable itself).